pudb/debugger.py

```diff
@@ -342,6 +342,22 @@ def set_frame_index(self, index):
 
         self.ui.stack_list._w.set_focus(self.ui.translate_ui_stack_index(index))
 
+    def open_file_to_edit(self, index):
+        self.curframe, line_number = self.stack[index]
+        filename = self.curframe.f_code.co_filename
+
+        editor = os.environ.get("EDITOR", "nano")
+        editor = editor.lower()
+
+        editor_command = f"{editor}"
```

Comment:
Could you move this to the `else` branch?

Reply:
Because of a request above this is not needed.

Every editor supports the `+l flag` so we can just `f"{editor} +l {line_number} {filename}"`